src/components/apply/ApplicantForm.js

```diff
@@ -20,25 +22,42 @@ const headingStyle = {
 
 const SectionHeading = ({ children }) => <div style={headingStyle}>{children}</div>;
 
-const ApplicantForm = ({ handleSubmit }) => (
+const ApplicantForm = ({ handleSubmit, addJointApplicant, history }) => (
   <div style={{marginBottom: 12}}>
-    <form onSubmit={handleSubmit((values) => console.log(values))} style={{display:'flex', flexDirection: 'column', alignItems: 'center'}}>
-      <SectionHeading>Your Identity</SectionHeading>
-      <IdentificationForm />
-      <SectionHeading>Contact Information and Address</SectionHeading>
-      <ContactInfo />
-      <Flex alignSelf='center' width='45%' style={{marginLeft: 80, marginBottom: 0, marginTop: 30}}>
-        <Field
-            name="addJointApplicant"
-            component={Checkbox}
-            label="Add joint applicant to my membership application"
-            labelStyle={{color: colors.basic}}
-            iconStyle={{fill: colors.basic, marginLeft: 10}}
-        />
-      </Flex>
-      <center><ContinueButton title='KEEP GOING' buttonProps={{type: 'submit'}} style={{ marginTop: 10 }}/></center>
-    </form>
-  </div>
+    <form onSubmit={handleSubmit((values) =>{
```

Comment:
What would be the perks of handling routing in redux vs using withRouter inside of the component?